app/views/ssh_keys/_ssh_keys_tab.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="tab-pane" id="ssh_keys">
+  <table class="<%= table_css_classes 'table-fixed' %>">
+    <% if user.ssh_keys.any? %>
+    <thead>
+      <tr>
+        <th width="40%"><%= _('Name') %></th>
+        <th width="40%"><%= _('Fingerprint') %></th>
+        <th width="80px"><%= _('Length') %></th>
+        <th width="20%"><%= _('Created') %></th>
+        <th width="100px"><%= _('Actions') %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% user.ssh_keys.each do |key| %>
+        <%= render 'ssh_keys/ssh_key_row', :f => f, :key => key %>
+      <% end %>
+    </tbody>
+  <% else %>
+    <tbody>
+      <tr>
+        <td class="blank-slate-pf">
```

Comment:
As discussed on IRC, this is unfortunately not possible as the welcome partial is only shown on the index controller. The ssh key list is only rendered as a tab on the users edit page.